app/views/shared/eligible_region_content_components/_professional_recognition.html.erb

```diff
@@ -80,23 +56,38 @@
   <% end %>
 <% end %>
 
-<% if region.sanction_check_none? %>
-  <p class="govuk-body">
-    You also need to show evidence of your work history.
-  </p>
+<% if region.sanction_check_written?%>
+  <%= render "shared/teaching_authority_contact_information", region: %>
+<% end %>
+
+<% if region.status_check_online? || region.sanction_check_online?%>
   <p class="govuk-body">
-    For each school this should include:
+    As your education department or authority has an online register of teachers,
+    you’ll also need to provide any reference numbers we need to check your record.
   </p>
-  <ul class="govuk-list govuk-list--bullet">
-    <li>school name</li>
-    <li>school city</li>
-    <li>school country</li>
-    <li>your job role</li>
-    <li>start date</li>
-    <li>end date</li>
-  </ul>
```

Comment:
We still need to show this under certain situations, specifically if the "work history" section isn't shown (because we haven't enabled the new feature yet). This is how it's handled in the view:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/6c84c19c956162ae103f783d4b079c5d2d14992b/app/views/shared/_eligible_region_content.html.erb#L50